src/services/edamamApi.js

```diff
@@ -1,28 +1,26 @@
-import {useState, useEffect} from "react"
+import { useState, useEffect } from "react";
 
-const GetRecipes = ()  => {
-    const [data, setRecipes] = useState([]);
-    const [query, setQuery] = useState("chicken");
-        
-    useEffect(() => {
-        const getRecipes =  async () => {
-            console.log("usao")
+export const GetRecipes = () => {
+  const [data, setRecipes] = useState([]);
```

Comment:
should we consider naming data to recipesData, since its more intuitive that way?